src/contexts/SearchContext.tsx

```diff
@@ -131,6 +137,9 @@ export const SearchContextProvider = ({ children }: Props) => {
       language: getQueryParam(urlParams, "language") as
         | SummaryLanguage
         | undefined,
+      style: getQueryParam(urlParams, "style") as
```

Comment:
I don't think we need lines 140-142 unless we also persist this value in the URL as a query parameter. The logic for doing so is defined here:

https://github.com/vectara/vectara-answer/blob/a46dd5181bb8bbfb6c8a95e3142372efbad7ee15/src/contexts/SearchContext.tsx#L216-L221

Reply:
Yes I think we do want to do it, just like language. So added this logic. Good catch.